pages/docs/couple-your-code/couple-your-code-waveform.md

```diff
@@ -0,0 +1,167 @@
+---
+title: Waveform iteration for time interpolation of coupling data
+permalink: couple-your-code-waveform.html
+keywords: api, adapter, time, waveform, subcycling, multirate
+summary: "With Waveform iteration you can interpolate your coupling data in time for higher order time stepping and more stable subcycling."
+---
+
+{% include warning.html content="These API functions are work in progress, experimental, and are not yet released. The API might change during the ongoing development process. Use with care." %}
+{% include note.html content="In this section, we will only discuss implicit coupling. We will, without loss of generality, only discuss the use of the API functions `readBlockVectorData` and `writeBlockVectorData` in the examples." %}
+
+preCICE allows the participants to use subcycling - meaning: to work with different time step sizes on their respective domains. If participants are using different timestep sizes, they have to synchronize at the end of each *time window*. If you want to know how this works and what a time window is, see ["Step 5 - Non-matching time step sizes" of the step-by-step guide](couple-your-code-timestep-sizes.html). In this section, we will take a closer look at the exchange of coupling data when subcycling, and advanced techniques for interpolation of coupling data inside of a time window.
+
+## Exchange of coupling data with subcycling
+
+preCICE only exchanges data and performs the coupling at the end of the last time step in the time window - the end of the time window. For the default case, preCICE only exchanges the data that was written at the very end of the time window. This approach automatically leads to discontinuities or "jumps" when going from one time windows to the next and, thus, lower accuracy (for details, see [here](https://onlinelibrary.wiley.com/doi/full/10.1002/nme.6443)).
```

Comment:
I'm experimenting with footnotes for external literature in 1bb5c4e. I think it's useful that the reference is on the same page. At the sametime I found always writing "see [out paper on ...](https://onlinelibrary.wiley.com/doi/full/10.1002/nme.6443)" quite repetitive.

It's still quite redundant to our [literature guide](https://precice.org/fundamentals-literature-guide.html) and especially the [preCICE publication list](https://precice.org/publications.html). I think a general solution to avoid duplicating references would be nice, but also more complicated and as far as I know we currently just duplicate references, if this is required.